checker/tests/nulless-conservative-defaults/annotatedfornullness/AnnotatedForNullness.java

```diff
@@ -52,4 +52,24 @@ void foo(@Initialized AnnotatedForNullness this) {
         annotatedForNullness(initializedField);
         annotatedForNullnessAndInitialization(initializedField);
     }
+
+    @AnnotatedFor("initialization")
+    void bar() {
+        // Expect an error because conservative defaults are applied to `unannotatedFor` and it
+        // expects a @FBCBottom @UnknownKeyFor @Nonull Object.
+        // ::error: (argument.type.incompatible)
+        unannotatedFor(initializedField);
+        // Do not expect an error because the warning is suppressed  other than initialzation
+        // hierarchy when conservative defaults are applied to source code.
+        // it expects an @Initialized @KeyForBottom @Nonull Object.
```

Comment:
There is some copy/paste problem here. Should this be one sentence? The second one starts with a lower case letter. The first one contains a double space, maybe where parts should be edited?

Reply:
Thanks for spoting it. I update the comments.